bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/ui/TabStartup.java

```diff
@@ -152,11 +157,35 @@ public void createControl(Composite parent) {
 			System.out.println("openocd.TabStartup.createControl() ");
 		}
 
+		if (!(parent instanceof ScrolledComposite))
+		{
+			ScrolledComposite sc = new ScrolledComposite(parent, SWT.V_SCROLL | SWT.H_SCROLL);
+			sc.setFont(parent.getFont());
+			sc.setExpandHorizontal(true);
+			sc.setExpandVertical(true);
+			sc.setShowFocusedControl(true);
+			this.createControl(sc);
+			Control control = this.getControl();
+			if (control != null)
+			{
+				sc.setContent(control);
+				sc.setMinSize(control.computeSize(SWT.DEFAULT, SWT.DEFAULT));
+				this.setControl(control.getParent());
+			}
+			return;
+
+		}
+
 		Composite comp = new Composite(parent, SWT.NONE);
 		setControl(comp);
 		GridLayout layout = new GridLayout();
 		comp.setLayout(layout);
-
+		EspConfigParser parser = new EspConfigParser();
+		String openOCDPath = new IDFEnvironmentVariables().getEnvValue(IDFEnvironmentVariables.OPENOCD_SCRIPTS);
+		if (!openOCDPath.isEmpty() && parser.hasBoardConfigJson())
+		{
+			createOpenOcdGroup(comp);
+		}
```

Comment:
Trying to understand why are we checking `openOCDPath` and `hasBoardConfigJson` to create this group?

Reply:
Hi @kolipakakondal,
for jtag flashing, we are using `program_esp_bins <path-to-build-dir> flasher_args.json verify exit` command, 
and if I remember correctly this command came to the openocd with the board config json. So if JSON config is in the openocd folder, then this command is available. I have replaced this code with `ESPFlashUtil.checkIfJtagIsAvailable()`, but it's basically the same for now. We can to this method the openocd version check to this method